tests/testthat/test-basic.R

```diff
@@ -33,5 +33,5 @@ test_that("can run the particle filter on the model", {
 
   ## TODO: this is not yet reliable, nor probably correct
   ## TODO: mcstate should spread out parameters like this for us by default
-  expect_type(pf$run(pars, pars, pars), "double")
+  expect_type(pf$run(pars), "double")
```

Comment:
Thanks - can you please update the DESCRIPTION file to refer to `mcstate (>= 0.2.0)` so that we always get consistent packages

Reply:
Done